form/form_test.go

```diff
@@ -229,6 +229,15 @@ func TestAppendTo(t *testing.T) {
 			"bar",
 		},
 
+		// Tests map with an empty value
+		{
+			"map[empty]",
+			&testStruct{Map: map[string]interface{}{
+				"empty": int64(0),
```

Comment:
Why are you not testing an empty string instead of `int64(0)` since you have a map of strings?

Reply:
Oops, good callout. I meant to write a comment about that and have added one in 886f389.

The explanation is that we are encoding these to a `form.Values` struct, and then in the test using `Get` to check a value against an expected a result. The problem is that `Get` returns an empty string for a value that wasn't set at all, which means that if we were using a string we wouldn't be able to differentiate a zeroed value from an empty value. As a cheap workaround, I've just used an `int64` instead. The behavior around how empties are handled is identical, but we can easily differentiate an empty integer once it's string encoded.